packages/inertia/src/events.js

```diff
@@ -0,0 +1,41 @@
+function fireEvent(name, options) {
```

Comment:
Ignore this, I accidentally added this js file (and removed on the following commit)